scripts/docker-develop/Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM adoptopenjdk:8
+FROM ubuntu:focal
+
+ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update -q && \
     apt-get upgrade -qq && \
+    apt-get install -y curl wget gnupg gnupg2 && \
```

Comment:
Why do we need to install gnupg here?

Reply:
`gnupg` is a pre-req of adoptopenjdk 11 hotspot. From the [docs](https://adoptopenjdk.net/installation.html#linux-pkg-deb):

> 1. Ensure the necessary packages are present:
>
>     sudo apt-get install wget apt-transport-https gnupg

This update of vsoch's test is passing in CI without the `apt-transport-https` but I'm happy to add that too for completeness?